tests/data/pz_pipeline_hsc.yaml

```diff
@@ -0,0 +1,27 @@
+description: |
+  Photo-z madness
+tasks:
+  pz_trainz:
+    class: lsst.meas.pz.estimate_pz_task_trainz.EstimatePZTrainZTask
+    config:
+      connections.pzModel: 'pzModel_trainz'
+      connections.pzEnsemble: 'pz_estimate_trainz'
+  pz_knn:
+    class: lsst.meas.pz.estimate_pz_task_knn.EstimatePZKNNTask
+    config:
+      connections.pzModel: 'pzModel_knn'
+      connections.pzEnsemble: 'pz_estimate_knn'
+subsets:
+  all_pz:
+    subset:
+      - pz_trainz
+      - pz_knn
+    description: |
+      All of the photoz algorithms
+# run me with
```

Comment:
We generally don't leave usage comments like this in pipeline files. You can put scripts in the `bin.src` if needed, though I'd say this is usage that a user should become familiar with from a tutorial elsewhere and not the first pipetask command they ever run.